extensions/ql-vscode/src/vscode-tests/cli-integration/run-remote-query.test.ts

```diff
@@ -180,15 +236,37 @@ describe('Remote queries', function() {
     }
   });
 
+  function verifyQlPack(qlpackPath: string, queryPath: string, packName: string, packVersion: string) {
+    const qlPack = yaml.safeLoad(fs.readFileSync(qlpackPath, 'utf8'));
+
+    // don't check the build metadata since it is variable
+    delete (qlPack as any).buildMetadata;
+
+    expect(qlPack).to.deep.equal({
+      name: packName,
+      version: packVersion,
+      dependencies: {
+        'codeql/javascript-all': '*',
+      },
+      library: false,
+      defaultSuite: [{
+        description: 'Query suite for remote query',
+        query: queryPath,
```

Comment:
The Windows tests are unhappy about path separators: https://github.com/github/vscode-codeql/runs/4259501142?check_suite_focus=true#step:9:224

Not sure if the problem comes from `queryPath` here, or from the logic in `run-remote-query.ts` itself....

Reply:
hmmmm...let me investigate.

Reply:
Made a fix. Glad we run these tests on multiple platforms.